src/main/java/seedu/teachbook/ui/ClassListPanel.java

```diff
@@ -47,4 +52,11 @@ protected void updateItem(Class classObj, boolean empty) {
         }
     }
 
+    public void reload() {
+        int N = ModelManager.getCurrentlySelectedClassIndex().getZeroBased();
```

Comment:
You can achieve this without using static. For example, use `logic.getCurrentlySelectedClassIndex` in `MainWindow.java` and pass the result to `reload()` as a parameter.